src/controllers/idp.ts

```diff
@@ -45,10 +48,22 @@ export async function resolveSelfCareIdentity(
   try {
     // ensure an apim account is created for every session,
     // so that every user operation can rely on that
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick(comment):** Be sure comments reflect the actual logic. This is a proposal:
‏
```suggestion
    // Ensure an APIM account exists for every session,
    // so that every user operation can rely on that.
    // If account creation is allowed for the first usage, the account is either fetcher or created
    // In such a case, the login fails when account creation fails.
    // Otherwise the account is fetched and the login fails when the account is not found
```

Reply:
284f890